roles/DCOS.agent_windows/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+- name: Windows | Defining DC/OS by using 'version' or, more specifically, 'image_commit'
+  set_fact:
+     dcos_version_specifier: "{{ dcos['version'] }}"
+     ansible_password: '{{ hostvars[inventory_hostname].pass }}'
```

Comment:
its called `ansible_password` not `pass`

Reply:
agreed, getting this from inventory